src/containers/my-resources/lessons-container/LessonsContainer.tsx

```diff
@@ -108,9 +108,24 @@ const LessonsContainer = () => {
     })
   }
 
+  const handleFetchData = useCallback(async () => {
+    await fetchData()
+  }, [fetchData])
+
+  useEffect(() => {
+    if (isError && error) {
```

Comment:
`isError` is just `Boolean(error)`, so you can omit it
```suggestion
    if (error) {
```

Reply:
Done.